SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Utils/AbstractDeviceInformationHelper.cs

```diff
@@ -74,13 +74,14 @@ protected virtual string GetLiveUpdatePackageHash()
 
         private string GetSdkVersion()
         {
-           return GetType().GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
+           return GetType().GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion;
```

Comment:
I am very concerned here. This field is mandatory. If we can't always have it we need to use a constant instead like in Xamarin.